images/debian-python/Dockerfile.debian

```diff
@@ -32,12 +32,20 @@ RUN apt-get update \
   && apt-get clean autoclean \
   && apt-get autoremove -y \
   && rm -rf /var/lib/apt/lists/* \
-  && rm -f /var/cache/apt/archives/*.deb \
-  && git clone https://github.com/pyenv/pyenv .pyenv \
+  && rm -f /var/cache/apt/archives/*.deb
```

Comment:
Split these as they are both long and it's a bit tedious having to run the whole thing again if you only change one side.